@ndlib/gatsby-theme-marble/src/components/Shared/ManifestImageGroup/ItemAlternateViews/AlternateImage/index.js

```diff
@@ -38,7 +44,9 @@ export const AlternateImage = ({
           />
           <Image
             src={findAltImage(allMarbleFile, index)}
-            alt={`Alternate View ${index}`}
+            alt={(!marbleItem.description && (classification.length > 1))
```

Comment:
This is the thing that is probably setting off the code climate.  
Turn this into a method that you call passing in marbleItem and I bet it will be fine.

Reply:
I took your advice and created `alt={(marbleItem.alttext()}` but I'm still getting called for a complexity issue. Do you think I should just make a component that creates the alttext and pass that into both of these components instead?